src/components/job-overview/step-summary.js

```diff
@@ -140,16 +140,16 @@ const StepSummary = ({ appName, jobName, step }) => (
         </Typography>
       </Link>
```

Comment:
Put a container element like a `<span>` around the link to prevent it from using the full width

![image](https://user-images.githubusercontent.com/79976935/130761563-829c9776-828b-472c-b472-a0770e61e43b.png)

Reply:
Added `justify-items: start` to `step-summary__description` class to override default value `stretch`